src/screens/HomeScreen/components/SearchCarousel/SearchCarousel.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import Dots from 'react-native-dots-pagination';
+import {Carousel} from '../../../../components';
+import {Pallete} from '../../../../theme';
+import {verticalScale, DeviceUtils} from '../../../../utils';
+import {SearchCarouselItem} from './SearchCarouselItem';
+import {SearchCarouselType} from './SearchCarousel.types';
+
+export const SearchCarouselRenderItem = ({index, item}: any) => {
+  const locale = DeviceUtils.locale;
+
+  return (
+    <SearchCarouselItem
+      name={
+        item.name[locale] !== undefined ? item.name[locale] : item.name.en_US
+      }
+      buttonText={item.buttonText}
+      image={item.image}
+      key={index}
+    />
+  );
+};
+
+const SearchCarousel = (props: SearchCarouselType) => {
+  const [currentSlideIndex, setCurrentSlideIndex] = useState(0);
+  const width = DeviceUtils.getDeviceWidth();
+  const {carouselData} = props;
+
+  return (
+    <View>
```

Comment:
Do we need this View? Can be React Fragment if View is not required

Reply:
No, It is working fine with fragment. So i removed the view.